### PR TITLE
Defining user-agent and GH action parameter version

### PR DESCRIPTION
**Pull request by gdelgadot**
Signed-off-by: Gloria Delgado <gloriad@vmware.com>

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -33,7 +37,7 @@ export const vibClient = clients.newClient({
   timeout: 10000,
   headers: {
     "Content-Type": "application/json",
-    "User-Agent": "vib-action/0.1-dev",
+    "User-Agent": `vib-action/${userAgentVersion}`,
```
Comment (recena) [inline src/main.ts:40]: I think you need to add also the double quote. I'm not sure.

**Review by recena — APPROVED** [review]

File: src/main.ts (hunk 1 above)
Comment (recena) [reply src/main.ts:40]: My mistake, I didn't review it properly.

File: src/main.ts [hunk 2]
```diff
@@ -14,6 +14,10 @@ const root =
     ? path.join(process.env.GITHUB_WORKSPACE, ".") // Running on GH but not tests
     : path.join(__dirname, "..") // default, but should never trigger
 
+const userAgentVersion = process.env.GITHUB_ACTION_REF
```
Comment (aruiz14) [inline src/main.ts:17]: Would this also work?

```suggestion
const userAgentVersion = process.env.GITHUB_ACTION_REF || "unknown";
```

File: src/main.ts [hunk 3]
```diff
@@ -14,6 +14,10 @@ const root =
...
+  ? process.env.GITHUB_ACTION_REF
+  : "unknown"
```
Comment (aruiz14) [inline src/main.ts:19]: Would this also work?

```suggestion
const userAgentVersion = process.env.GITHUB_ACTION_REF || "unknown"
```

Comment (gdelgadot) [reply src/main.ts:19]: TBH I don't know 😅

Comment (gdelgadot) [reply src/main.ts:19]: I will keep the first option `?:`, since it has been used in other parts of the code as well :)

**Pull request merged**